pkg/etcdenvvar/etcd_env.go

```diff
@@ -243,6 +243,24 @@ func getHardwareSpeedValues(envVarContext envVarContext) (map[string]string, err
 	return hwspeedhelpers.HardwareSpeedToEnvMap(speed)
 }
 
+func getEtcdDBSize(envVarContext envVarContext) (map[string]string, error) {
+	etcd, err := envVarContext.etcdLister.Get("cluster")
+	if err != nil {
+		return nil, fmt.Errorf("failed to retrieve etcd CR: %v", err)
+	}
+
```

Comment:
okay that's very nitpicky, but how about:

```suggestion

   backendQuotaGiB := int64(8)
   if etcd.Spec.BackendQuotaGiB != 0 {
      backendQuotaGiB = int64(etcd.Spec.BackendQuotaGiB)
   }
   
   return map[string]string{
		"ETCD_QUOTA_BACKEND_BYTES": GibibytesToBytesString(backendQuotaGiB),
   }, nil 
```

then the below logic can just be int64 based, so you don't have to worry about casting the values up or down anymore.

Reply:
lgtm :D :D .. yes way easier to follow 👍🏽 💯

Reply:
fixed 👍🏽